ppx_const.opam

```diff
@@ -1,10 +1,10 @@
 # This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
 version: "2.0"
-synopsis: "Compile-time \"if\" statement for conditional inclusion of code"
+synopsis: "Compile-time \"if\" statement for conditional inclusion of code."
 description: """
 This is a ppx extension which adds `if#const` and `match#const` constructs to
-OCaml. They behave like normal `if` and `match`, but conditions are evaluated
+OCaml. They behave like normal `if` and `const`, but conditions are evaluated
```

Comment:
Your changes here (the removed period and the substitution of "match" for "const" are) incorrect. These are typos that were fixed in the most recent release, you have reintroduced them. Please revert these changes.

Reply:
Hmm I was first surprised to see those changes as I didn't rewrite these parts myself. It turns out your `dune-project` file's package description is desynchronized from your opam file.

I think the proper fix would be to update your `dune-project` instead, I'll take care of it!